sleap_nn/training/model_trainer.py

```
@@ -568,10 +573,22 @@ def __init__(
+        # Pre-trained weights for the encoder stack - only for swint and convnext
         if self.model_config.pre_trained_weights:
             self.model.backbone.enc.load_state_dict(ckpt, strict=False)
 
+        # Initializing model (encoder + decoder) with trained ckpts
+        # TODO: Handling different input channels
+        if trained_ckpts_path is not None:
+            print(f"Loading weights from `{trained_ckpts_path}` ...")
+            ckpt = torch.load(trained_ckpts_path)
+            ckpt["state_dict"] = {
+                k: ckpt["state_dict"][k]
+                for k in ckpt["state_dict"].keys()
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)